data/campaigns/Eastern_Invasion/ART_MUSIC_LICENSE.txt

```diff
@@ -0,0 +1,303 @@
+
+
+MISC
+---------------
+
+arcane-icon
+crystal-quiver-icon
+herb-icon
+leader-gold
+plague-staff-icon
+potion-of-vitality-icon
+ring-of-invisibility-icon
+sentinel-icon
+soulsword-icon
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+
+PORTRAITS
+---------------
+
+gaennell-adept
+mal-ravanal-ghost
+pyre-wight
+terraent
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+remainder is pre-existing wesnoth content (GPL v2.0 or later)
+
+
+PROJECTILES
+---------------
+
+shadow-blast - from the Liberty campaign (GPL v2.0 or later)
+
+
+SCENERY
+---------------
+
+blood
+bookshelf
+gore
+    from After The Storm, used with Iris's permission  (GPL v2.0 or later)
+
+cave_entrance
+cave_entrance_empty
+dwarven-keep-tile
+rune_line
+rune_line_glow
+rune_line_glow_base
+statue-delfador
+statue-haldric
+throne
+throne-small
+    made by dalas by modifying pre-existing wesnoth images  (GPL v2.0 or later)
+
+
+TERRAIN
+---------------
+
+terrain_type_urban
+terrain_type_urban_30
+    made by dalas (CC BY-SA 4.0)
+
+remainder is original wesnoth content modified by dalas (GPL v2.0 or later)
+
+
+WELDYN
+---------------
+
+walls_S15
+walls_S18b
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+
+
+
+
+################################################################################################################################################################################
+# UNIT IMAGES
+################################################################################################################################################################################
+
+ANT
+---------------
+
+egg sac
+egg sac remains
+    made by dalas (CC BY-SA 4.0)
+
+queen ant
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+
+BANNERET
+---------------
+
+base image from The_Gnat's Reign of the Lords add-on  (GPL v2.0 or later)
+
+animations
+    by dalas (GPL v2.0 or later)
+
+
+DACYN
+---------------
+
+all images
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+
+NAGA
+---------------
+
+base image from The_Gnat's Reign of the Lords add-on (GPL v2.0 or later)
+
+animations
+    by dalas (GPL v2.0 or later)
+
+
+OGRES
+---------------
+
+all images from The_Gnat's Reign of the Lords add-on (GPL v2.0 or later)
+
+
+OWAEC
+---------------
+
+all images
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+
+ROYAL WARRIOR
+---------------
+
+all images
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+
+SHADOW MAGE
+---------------
+
+all images
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+
+TERRAENT
+---------------
+
+all images
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+
+TOWNSFOLK
+---------------
+
+https://units.wesnoth.org/1.14/Era_of_Myths/en_US/EOM.html
```

Comment:
@stevecotton These are the other main ones to look at, as far as I can tell they are all under GPL but are we okay including these or should I redraw the sprites?

Reply:
I can't work work out which of the townsfolk come from the EoM. The nobles definitely have something in common with EoM's vampires, but I'd guess it's a common ancestor.

Artistically, I'd guess that they're all based on our GPL'd images, but have no idea who should be credited for the revisions.